draft-ietf-jsonpath-base.md

```diff
@@ -227,7 +220,7 @@ Nodelist:
   While this list can be represented in JSON, e.g. as an array, the
   nodelist is an abstract concept unrelated to JSON values.
 
-Normalized Path:
```

Comment:
I definitely prefer 'Normalized Path'.

> "Normalization is the process of organizing a database to reduce redundancy and improve data integrity.
Normalization also simplifies the database design so that it achieves the optimal structure composed of atomic elements (i.e. elements that cannot be broken down into smaller parts)." 

https://database.guide/what-is-normalization/

>  In this context, a canonical form is a representation such that every object has a unique representation.

https://en.wikipedia.org/wiki/Canonical_form#Computing

So 'normalized path' in its canonical form is much more concrete than merely 'output path' . Its syntax definition is:

```
normalized-path = root-selector *("[" (quoted-member-name / positiv-element-index) "]")

positive-element-index = ( "0" / (DIGIT1 *DIGIT) )
```

Reply:
A 'normalized path' usually is an 'output path'. It is frequently used as 'json-path' for revisiting previous selections (post processing ?) and is understood natively as accessor at least in JavaScript.

Reply:
Fair enough. I'm going to try using normalized path throughout. Please see the second commit.

Reply:
hmm ... `quoted-member-name` must be replaced by `single-quoted-member-name` above.

Reply:
> hmm ... `quoted-member-name` must be replaced by `single-quoted-member-name` above.

Indeed, but that is not everything we need to go fully canonical (https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/152#discussion_r796785776).